packages/nimble-components/src/wafer-map/styles.ts

```diff
@@ -1,3 +1,57 @@
 import { css } from '@microsoft/fast-element';
+import { borderColor, borderWidth } from '../theme-provider/design-tokens';
 
-export const styles = css``;
+export const styles = css`
+    .wafer-map-container {
+        width: 500px;
+        height: 500px;
+        position: relative;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+
+    .svg-root {
+        width: inherit;
+        height: inherit;
+    }
+
+    .top {
+        transform: rotate(-90deg);
+    }
+
+    .right {
+        transform: rotate(0deg);
+    }
+
+    .left {
+        transform: rotate(180deg);
+    }
+
+    .bottom {
+        transform: rotate(90deg);
+    }
```

Comment:
In order to emphasize that these are coupled to the .svg-root you can use a [double class selector](https://css-tricks.com/multiple-class-id-selectors/#aa-double-class-selector):

```suggestion
    .svg-root.top {
        transform: rotate(-90deg);
    }

    .svg-root.right {
        transform: rotate(0deg);
    }

    .svg-root.left {
        transform: rotate(180deg);
    }

    .svg-root.bottom {
        transform: rotate(90deg);
    }
```

A benefit of the above is that by reading the css it is more apparent which element the classes belong to.